conf/azurebatchdev.config

```diff
@@ -0,0 +1,71 @@
+//Nextflow config file for running on Azure batch
+params {
+  config_profile_description = 'Azure BATCH Dev Cloud Profile'
+  config_profile_contact = 'Venkat Malladi (@vsmalladi)'
+  config_profile_url = 'https://azure.microsoft.com/services/batch/'
+
+  // Active Directory
+  principal_id      = null
+  principal_secret  = null
+  tenant_id         = null
+
+  // Storage
+  storage_name      = null
+
+  // Batch
+  az_location       = "westus2"
+  batch_name        = null
+
+  vm_type           = "Standard_D8s_v3"
+  autopoolmode      = true
+  allowpoolcreation = true
+  deletejobs        = true
+  deletepools       = true
+  az_worker_pool    = "auto"
+
+  // ACR
+  acr_registry      = null
+  acr_username      = null
+  acr_password      = null
+
+}
+
+process {
+    executor = "azurebatch"
+}
+
+azure {
+  process {
+    queue = params.az_worker_pool
+  }
+  activeDirectory {
+    servicePrincipalId = params.principal_id
+    servicePrincipalSecret = params.principal_secret
+    tenantId = params.tenant_id
+  }
+  storage {
+      accountName = params.storage_name
+  }
+  batch {
+      location                = params.az_location
+      accountName             = params.batch_name
+      tokenDuration           = "24h"
+      autoPoolMode            = params.autopoolmode
+      allowPoolCreation       = params.allowpoolcreation
+      deleteJobsOnCompletion  = params.deletejobs
+      deletePoolsOnCompletion = params.deletepools
+      pools {
+          auto {
+              vmType    = params.vm_type
+              autoScale = true
+              vmCount    = 1
+              maxVmCount = 12
+          }
```

Comment:
I was thinking that we agreed on using multiple static pools to have a stable testing env 🤔

Reply:
Agreed forgot about that.  Will Change